src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/repositories/ResourceContainerRepository.kt

```diff
@@ -103,7 +110,7 @@ class ResourceContainerRepository(
         private val helpContentTypes = listOf(ContentType.TITLE, ContentType.BODY)
         private val dublinCoreIdDslVal = DSL.`val`(dublinCoreId)
 
-        fun import(node: TreeNode) {
+        fun import(node: OtterTreeNode<Any>) {
```

Comment:
Should Any change to CollectionOrContent?

Reply:
Yes.  Fixed now.